conans/client/tools/win.py

```diff
@@ -268,18 +267,18 @@ def vcvars_command(settings, arch=None, compiler_version=None, force=False, vcva
         if not vs_path or not os.path.isdir(vs_path):
             raise ConanException("VS non-existing installation: Visual Studio %s" % str(compiler_version))
         else:
-            vcvars_path = ""
             if int(compiler_version) > 14:
                 vcvars_path = os.path.join(vs_path, "VC/Auxiliary/Build/vcvarsall.bat")
                 command = ('set "VSCMD_START_DIR=%%CD%%" && '
                            'call "%s" %s' % (vcvars_path, vcvars_arch))
-                if winsdk_version:
-                    command += " %s" % winsdk_version
-                if vcvars_ver:
-                    command += " -vcvars_ver=%s" % vcvars_ver
             else:
                 vcvars_path = os.path.join(vs_path, "VC/vcvarsall.bat")
                 command = ('call "%s" %s' % (vcvars_path, vcvars_arch))
+        if int(compiler_version) > 12:
```

Comment:
Confirmed version?

Reply:
But 2015 is 14. Why 12?

Reply:
previous version to 14 was 12. At least this is what `cmake --help` says

Reply:
ah! ok, > 12. Sorry

Reply:
Why don't ">=14"?

Reply:
yes, you are right!